docs/diagrams/SummariseClassDiagram.puml

```diff
@@ -8,8 +8,12 @@ abstract class "{abstract}\nCommand" {
 }
 
 interface Model <<interface>> {
-    + addPerson(Person)
-    + hasPerson(Person) : boolean
+    /' Fields '/
+    +PREDICATE_SHOW_ALL_PERSONS : Predicate
+
```

Comment:
I believe there should be a dependency of the piechartwindow on the summarisecommand since the former is getting the data from the latter,

Reply:
Hi, as this diagram focuses on just summarise class, I do not think we need to show the dependency of PieChartWindow here.